src/main/src/main/java/org/geoserver/config/impl/GeoServerInfoImpl.java

```diff
@@ -47,6 +47,8 @@
     protected Boolean xmlExternalEntitiesEnabled = Boolean.FALSE;
     
     protected String lockProviderName;
+
+    protected WebUIMode webUIMode = WebUIMode.DEFAULT;
```

Comment:
Copyright header not updated. There are shell scripts to automatically update the headers that have been shared on the list multiple times, why don't you use them instead of checking manually file by file, and risking of losing an update?
Just commit, run the following (it's called fix_headers on my system), commit again and squash the commit. Only needs to be done once before making a pull request, minimum overhead:

````
sed -i 's/\(([cC])\) 2014 Open Source Geospatial Foundation/\1 2014 - 2016 Open Source Geospatial Foundation/' $(git diff-tree --no-commit-id --name-only -r HEAD)
sed -i 's/\(([cC])\) 2014-2015 Open Source Geospatial Foundation/\1 2014 - 2016 Open Source Geospatial Foundation/' $(git diff-tree --no-commit-id --name-only -r HEAD)
sed -i 's/\(([cC])\) 2014 - 2015 Open Source Geospatial Foundation/\1 2014 - 2016 Open Source Geospatial Foundation/' $(git diff-tree --no-commit-id --name-only -r HEAD)
````